.gitignore

```diff
@@ -26,4 +26,5 @@
 **/pb/*.pb.go
 **/pb/*.pb.gw.go
 **/pb/*.swagger.json
-database/data
\ No newline at end of file
+database/data
+app/state
```

Comment:
remove this one but don't commit any blobs from here

Reply:
I see it was already merged, did you make the change?